datasets/gov/vat/saugomos_veisles.csv

```diff
@@ -0,0 +1,74 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vat/saugomos_veisles,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Veisle,,,id,,,4,open,,"Pateikiama informacija apie į Nacionalinį sąrašą įrašytas veisles, išbrauktas veisles, veislių palaikytojų ir selekcininkų įgaliotųjų atstovų sąrašus.",
+,,,,,id,string,,,,4,open,,Unikalus identifikatorius,
+,,,,,grupe,string,,,,4,open,,Augalų grupė,
+,,,,,,,,,"""ŽEMĖS ŪKIO AUGALAI""",,,,,
+,,,,,,,,,"""DARŽOVĖS""",,,,,
+,,,,,,,,,"""Sodo augalai""",,,,,
+,,,,,,,,,"""Dekoratyviniai augalai""",,,,,
+,,,,,gentis,string,,,,4,open,,Augalų genčių ir rūšių grupė,
+,,,,,,,,,"""Javai""",,,,,
+,,,,,,,,,"""Aliejiniai ir pluoštiniai augalai""",,,,,
+,,,,,,,,,"""Runkeliai""",,,,,
+,,,,,,,,,"""Pašariniai pupiniai augalai""",,,,,
+,,,,,,,,,"""Pašariniai migliniai augalai""",,,,,
+,,,,,,,,,"""Daržovės""",,,,,
+,,,,,,,,,"""Bulvės""",,,,,
+,,,,,,,,,"""Serbentai""",,,,,
+,,,,,,,,,"""Trešnės, Vyšnios""",,,,,
+,,,,,,,,,"""Braškės, žemuogės""",,,,,
+,,,,,,,,,"""Obelys""",,,,,
+,,,,,,,,,"""Pašariniai kitų rūšių augalai""",,,,,
+,,,,,,,,,"""Kriaušės""",,,,,
+,,,,,veisle,string,,,,3,open,,Augalo veislės pavadinimas,
+,,,,,rusis,string,,,,2,open,,Augalo rūšis,
+,,,,,atnaujinimo_metai,integer,,,,2,open,,Įrašymo atnaujinimo metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,galioja_iki,integer,,,,2,open,,"Metai, iki kurių gruodžio 31 d. galioja įrašymas",
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,atstovas,string,,,,2,open,,Atstovo pavadinimas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"", type: ""ref"", ref: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"")	",4,,"spinta:205, manifest:1290",,
+,,,,,atstovo_adresas,string,,,,1,open,,Atstovo adresas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"", type: ""ref"", ref: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"")	",4,open,"spinta:205, manifest:1290",,
+,,,,,metai,integer,,,,2,open,,Įrašymo metai,
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,palaikytojo_nr,string,,,,4,open,,Veislės palaikytojo Nr.,
+,,,,,palaikytojas,string,,,,2,open,,Palaikytojo pavadinimas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"", type: ""ref"", ref: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"")	",4,,"spinta:205, manifest:1290",,
+,,,,,pastabos,string,,,,2,open,,Pastabos,
+,,,,,buveine,string,,,,1,open,,Palaikytojo adresas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"", type: ""ref"", ref: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"")	",4,open,"spinta:205, manifest:1290",,
```

Comment:
Prašau įvertinti, ar šis komentavimas reikalingas.

Reply:
Kaip suprantu, jūs siūlote ištrinti komentarą, nes adresai yra ne tik LT juridinių asmenų?